scene/gui/color_picker.cpp

```diff
@@ -43,6 +43,340 @@
 #include "thirdparty/misc/ok_color.h"
 #include "thirdparty/misc/ok_color_shader.h"
 
+String ColorMode::get_slider_label(int idx) {
+	if (idx > labels.size() - 1) {
```

Comment:
This should probably use `ERR_FAIL_INDEX`, which will also check for negative values